docs/doc/30-reference/20-functions/50-aggregate-functions/aggregate-retention.md

```diff
@@ -0,0 +1,60 @@
+---
+title: RETENTION
+---
+
+Aggregate function
+
+The RETENTION() function takes as arguments a set of conditions from 1 to 32 arguments of type UInt8 that indicate whether a certain condition was met for the event.
+
+Any condition can be specified as an argument (as in WHERE).
+
+The conditions, except the first, apply in pairs: the result of the second will be true if the first and second are true, of the third if the first and third are true, etc.
+
+## Syntax
+
+```
+RETENTION(cond1, cond2, ..., cond32);
+```
+
+## Arguments
+
+| Arguments   | Description |
+| ----------- | ----------- |
+| cond  | An expression that returns a Boolean result |
+
+## Return Type
+
+The array of 1 or 0.
+
+## Examples
+
+```
+CREATE TABLE retention_test(date Date, uid Int32) ENGINE = Memory;
```

Comment:
Let's use the sql style name.

https://github.com/datafuselabs/databend/blob/d3c0139e18cc98b97a82735df28f6693bbc85f70/docs/doc/30-reference/10-data-types/index.md

Reply:
ok